packages/circuit-ui/util/key-codes.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+/**
+ * `event.keyCode` has been deprecated and replaced by `event.key`, however,
+ * not all browsers implement it yet.
+ * TypeScript assumes that `event.key` is always available, so it complains
+ * about `event.keyCode` never being available.
+ */
+
+export const isEnter = (event: KeyboardEvent | React.KeyboardEvent): boolean =>
+  // @ts-expect-error `keyCode` is needed to support older browsers.
+  'key' in event ? event.key === 'Enter' : event.keyCode === 13;
+
+export const isEscape = (event: KeyboardEvent | React.KeyboardEvent): boolean =>
+  // @ts-expect-error `keyCode` is needed to support older browsers.
```

Comment:
Can we remove this after we drop support for IE etc.? Maybe we can add a FIXME for it somewhere (there's already one in the ImageInput for using the brightness() css filter)

Reply:
Yes, we'll be able to drop it then. I'll add a comment 👍🏻

Reply:
Done in 067a3e5d4.